scipy/_lib/tests/test__util.py

```diff
@@ -10,9 +10,9 @@
 from pytest import raises as assert_raises
 import hypothesis.extra.numpy as npst
 from hypothesis import given, strategies, reproduce_failure  # noqa: F401
-from scipy.conftest import array_api_compatible
+from scipy.conftest import array_api_compatible, skip_array_api_invalid_arg
 
-from scipy._lib._array_api import xp_assert_equal
+from scipy._lib._array_api import xp_assert_equal, is_numpy, copy as xp_copy
```

Comment:
I'm not opposed to changing the names of the helpers themselves to `xp_copy` and `xp_size` if that would be easier than having these aliases everywhere. `size` is just exposed from `array_api_compat` (no wrapper), and `copy` was written by Andrew.